plugins/techdocs-backend/config.d.ts

```diff
@@ -24,7 +24,7 @@ export interface Config {
      * attr: 'storageUrl' - accepts a string value
      * e.g. storageUrl: http://localhost:7000/api/techdocs/static/docs
      */
-    storageUrl: string;
+    storageUrl?: string;
```

Comment:
i wonder if you can add a `@deprecated` annotation on this one

Reply:
Yes, that's a good idea.